sympy/functions/combinatorial/factorials.py

```diff
@@ -32,7 +32,12 @@ def _eval_simplify(self, **kwargs):
 ###############################################################################
 ######################## FACTORIAL and MULTI-FACTORIAL ########################
 ###############################################################################
-
+class multinomial(Function):
+        def multinomial(n, m):
```

Comment:
You should put this into `eval`

Reply:
And I think that this should be taken as a n-ary function

Reply:
Okay, sir. Working on it.

Reply:
> And I think that this should be a function that should take variable number of arguments

Sir, do you mean make an array of variable arguments?